pkgdb/src/nix-state.cc

```diff
@@ -35,6 +35,8 @@ initNix()
   // NOLINTNEXTLINE
   nix::setStackSize( ( std::size_t( 64 ) * 1024 ) * 1024 );
   nix::initNix();
+  /* Set the GC to handle forking properly */
```

Comment:
Suggestion: If this is a Boehm setting then call that out specifically since this is mostly Nix setup